src/components/DocLayout/SidebarMenu/index.js

```diff
@@ -76,14 +76,18 @@ export default function SidebarMenu({ id, sidebar, currentPath, onClick }) {
       })
     }
 
-    const node = document.getElementById(currentPath)
+    const node = document.getElementById(currentPath.replace(/\/$/, ''))
```

Comment:
This should be handled at the serverside I believe. It should act as the gatekeeper which prevents us from having to deal with the trailing-slash-case on the client.

Reply:
We can add a redirect to redirects-list.json which redirects from `^(.+)/$` to `$1` and that should do it. We never want trailing slashes.

@iAdramelk told me this breaks the static server even if its [redirects](https://github.com/expressjs/serve-static#redirect) option is disabled, but we can always use [serve-handler](https://npmjs.com/serve-handler). The choice doesn't matter much. This is all going in S3 eventually.

Reply:
I'll take a look a `serve-handler` tomorrow.
But we also discussed this issue [yesterday at Slack](https://evilmartians.slack.com/archives/CLN9J8Q8K/p1584377665016000?thread_ts=1584348448.007700&cid=CLN9J8Q8K) and decided that it can be resolved after merging with blog's codebase.